pom.xml

```diff
@@ -6,11 +6,11 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.7.5</version>
+        <version>3.0.4</version>
     </parent>
     <groupId>com.obsidiandynamics.kafdrop</groupId>
     <artifactId>kafdrop</artifactId>
-    <version>3.31.0-SNAPSHOT</version>
+    <version>3.32.0-SNAPSHOT</version>
```

Comment:
Given the importance of this change, maybe this require a new major version? What do you think?

cc @ekoutanov

Reply:
Also maybe you can rename the PR title to something like: "Update to JDK 17 and SpringBoot 3" ?

Reply:
Title makes sense to me! @davideicardi

Reply:
Yes, 4.0.0!

(I need to first publish the current release, then I will merge this PR and publish a 4.0 release)